client/src/game-service.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import { Game } from './db-types';
```

Comment:
Move GameService and ReviewService to separate files so they can be mocked for testing.